src/Build.UnitTests/BackEnd/RedirectConsoleWriter_Tests.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+//
+
+using System;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Build.Execution;
+using Xunit;
+
+namespace Microsoft.Build.Engine.UnitTests.BackEnd
+{
+    public class RedirectConsoleWriter_Tests
```

Comment:
minor: are msbuild test classes usually sealed?

Reply:
They sometimes are and sometimes aren't. Seems to be correlations within assemblies, and this one mostly is not sealed, but I don't think it hurts.